web/styles.css

```diff
@@ -651,6 +680,26 @@ nav.menu li {
     color: #ccc;
 }
 
+/*
+We have to do something with the scrollbars as they flicker due to DOM
```

Comment:
Do we want this for all of our scrollbars or just the inspector ones?

Reply:
this impacts the infinite scrolling table as well so yes.
We can tweak this style so we still fade in and out on hover but I think this is at least a non-flickery default.